pkg/process/monitor/process_monitor.go

```diff
@@ -293,9 +295,8 @@ func (pm *ProcessMonitor) Initialize() error {
 					return
```

Comment:
shouldn't we call CloseDone here as well?

Reply:
I added for all `if !ok return ` pattern